src/tir/ir/expr.cc

```diff
@@ -45,24 +45,28 @@ SizeVar::SizeVar(std::string name_hint, DataType t)
 SizeVarNode::SizeVarNode(DataType t, std::string name_hint)
     : VarNode(t, std::move(name_hint)) {}
 
-IterVar IterVarNode::make(Range dom,
-                          Var var,
-                          IterVarType t,
-                          std::string thread_tag) {
-  ObjectPtr<IterVarNode> n = make_object<IterVarNode>();
-  n->dom = dom;
-  n->var = var;
-  n->iter_type = t;
-  n->thread_tag = thread_tag;
-  return IterVar(n);
+IterVarNode::IterVarNode(DataType dtype, std::string name_hint,
```

Comment:
I'm not sure whether we need a constructor for Node

Reply:
it's used in https://github.com/apache/incubator-tvm/pull/4765/files#diff-b0d470d0765d89448d8d18506ba0c54eR61